module-workers.tf

```diff
@@ -65,6 +65,7 @@ module "workers" {
   volume_kms_key_id     = var.worker_volume_kms_key_id
   worker_nsg_ids        = concat(var.worker_nsg_ids, [module.network.worker_nsg_id])
   worker_subnet_id      = lookup(module.network.subnet_ids, "workers")
+  preemptible           = var.worker_preemptible
```

Comment:
`worker_preemptible_config`/`preemptible_config` for root/submodule variables may be a little more descriptive, from the core instance variable "[preemptible_instance_config](https://registry.terraform.io/providers/oracle/oci/latest/docs/resources/core_instance_configuration#preemptible_instance_config)".